cypress/integration/immutableDatabaseTests/services.ts

```diff
@@ -153,7 +151,7 @@ describe('Dockstore Home', () => {
     cy.contains('tr', 'Git Reference');
     cy.contains('td', '1.3');
     cy.contains('tr', 'Date Modified');
-    cy.contains('td', 'Jul 19, 2019, 1:13:48 PM');
```

Comment:
I think we can lose the seconds, but lets keep minutes. 
When debugging, I did often upload a few versions rapidly and it can be good to distinguish between them. 
Also while at it, I prefer YYYY-MM-DD